### DISCUSSION TITLE
NCAIO on Synology Docker - Domainchecker error Port 443

### DISCUSSION THREAD

**Discussion by TheNextclouder** [nextcloud/all-in-one#4935] [Questions] [answered]
Hello

When I try to install NCAIO then i get stuck at "New AIO instance" providing my domain.
(Im installing over the internet)    cloud.xxxxxxxx.com
Following error pops up:
`The domain is not reachable on Port 443 from within this container. Have you opened port 443/tcp in your router/firewall? If yes is the problem most likely that the router or firewall forbids local access to your domain. You can work around that by setting up a local DNS-server.`

Reverse Proxy does 443 to 8448

YAML:
![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/08acd950-d9f7-432a-9bd6-83e26b2267ea)

Apache of domain checker:
![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/952805bf-5986-48a0-a6ec-2bc291bc532c)

Can someone help me to solve this?
Thanks

**Comment by TheNextclouder** [comment]
Just tried to do the same from within the network, but i still got the same error....

**Comment by szaimen** [comment]
Hi, can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things?

**Comment by TheNextclouder** [comment]
This is how my YAML looks like after I restarted from scratch and Port 11000
>    ports:
>       #- 9080:80 # Can be removed when running behind a web server or reverse proxy (like Apache, Nginx, Cloudflare Tunnel and else). 
>       - 8448:8080
>       #- 8443:8443 # Can be removed when running behind a web server or reverse proxy (like Apache, Nginx, Cloudflare Tunnel and else). 
>     environment: # Is needed when using any of the options below
>       #AIO_DISABLE_BACKUP_SECTION: true # Setting this to true allows to hide the backup section in the AIO interface. See 
>       APACHE_PORT: 11000 # Is needed when running behind a web server or reverse proxy (like Apache, Nginx, Cloudflare Tunnel and else). See 
>       APACHE_IP_BINDING: 0.0.0.0 # Should be set when running behind a web server or reverse proxy (like Apache, Nginx, Cloudflare Tunnel and else) that is running on the same host. 
> 

![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/88a2e910-bed8-403b-9c49-cbc44ac59d88)

However point 7:
Check if after the mastercontainer was started, the reverse proxy if running inside a container, can reach the provided apache port. You can test this by running nc -z localhost 11000; echo $? from inside the reverse proxy container. If the output is 0, everything works. Alternatively you can of course use instead of localhost the ip-address of the host here for the test.

Executed from with mastercontainer console:
`nc -z localhost 11000; echo $?`   fails
`nc -z 192.168.1.xxx 11000; echo $?` works

Keep in mind, my reverse proxy is the Synology NAS.

Disabled the firewall briefly, makes no difference...

**Comment by szaimen** [comment]
> nc -z 192.168.1.xxx 11000; echo $? works

Indeed this is where the reverse proxy needs to point at

**Reply by TheNextclouder** [reply]
but where do i have to put this values in?
443 to 11000 HTTP ?

**Reply by szaimen** [reply]
into your synology reverse proxy

**Reply by TheNextclouder** [reply]
![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/1d750811-a22d-425a-be5e-917566a2ddeb)
![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/5cc8c361-4026-480e-9af0-bdbb2ef13079)
Still not working that way....

**Reply by szaimen** [reply]
> Still not working that way....

What exactly does not work?

**Reply by TheNextclouder** [reply]
![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/9d50a47f-21d3-461e-a383-5b95138ba56b)
When I type in the URL in the new Instance, i get the following error:
> The domain is not reachable on Port 443 from within this container. Have you opened port 443/tcp in your router/firewall? If yes is the problem most likely that the router or firewall forbids local access to your domain. You can work around that by setting up a local DNS-server.

**Reply by szaimen** [reply]
You might want to use this then: https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-skip-the-domain-validation

**Reply by TheNextclouder** [reply]
Ok, I can try.
Wouldnt the domain/this step be needed for using NextCloud AIO over the internet?
Thanks so far

**Reply by TheNextclouder** [reply]
Now i skipped the domain validation, but there are new troubles...

> Start container nextcloud-aio-nextcloud failed: {"message":"error gathering device information while adding custom device \"/dev/dri\": no such file or directory"}.

**Reply by szaimen** [reply]
Did you see the warning here? https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-enable-hardware-transcoding-for-nextcloud

**Reply by TheNextclouder** [reply]
![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/9e889881-9005-4c6d-9e30-a3b60b5162aa)
Aside that it seems some stuff is not even running...

**Reply by szaimen** [reply]
Yes, the reason is that you incorrectly configured this value even though this device does not exist on your host. You should now disable it, via the env, recreate the mastercontainer and afterwards restart the containerd via the aio interface

**Reply by TheNextclouder** [reply]
Already done, no more complaints about "/dev/dri" and still issues

**Reply by szaimen** [reply]
> still issues

What issues?

**Reply by TheNextclouder** [reply]
Ok, im a bit futher.
![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/e42afc73-14cd-4ebb-b571-25202d7ad2d3)
![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/c8d05964-66d1-446f-951f-958dd73aac2e)

However if I open nextcloud:
![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/97f21efa-e414-43eb-a6e7-ae5fc5b8d13b)

**Reply by szaimen** [reply]
Can you show a screenshot of the current synology reverse proxy setup again?

**Reply by TheNextclouder** [reply]
Of course
![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/ea91b1ac-b1b3-40fe-9c2c-e93b3652212c)
![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/641be3bf-7b83-4544-874c-6bbb2f95506f)

![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/bc2bac7c-50ff-4a39-a829-94f0af87bbae)

Do I need more RP entries?
For Port 8443 i.e.?

Thanks

**Reply by szaimen** [reply]
Looks fine. I suspect now maybe a firewall issue. Can you check https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-run-aio-on-synology-dsm for some hints?

**Reply by TheNextclouder** [reply]
> 2024-07-11T08:59:32.762782782Z nc: getaddrinfo for host "nextcloud-aio-onlyoffice" port 80: Try again
> 2024-07-11T08:59:32.762941884Z waiting for OnlyOffice to become available...

![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/a63b5de9-e1c5-4cac-9e71-a015df129f0d)

Might that be the issue, that the nextcloud container is waiting for onlyoffice?

**Reply by szaimen** [reply]
Can you post the nextcloud container logs here?

**Reply by TheNextclouder** [reply]
[NextcloudLog20240711_1145.txt](https://github.com/user-attachments/files/16174577/NextcloudLog20240711_1145.txt)

**Reply by szaimen** [reply]
Yes looka like it is waiting for onlyoffice. Can you disable onlyoffice and try again?

**Reply by TheNextclouder** [reply]
![grafik](https://github.com/nextcloud/all-in-one/assets/174714471/a1464339-68df-4081-81b1-775482a352f1)
Still the same, it waits on onlyoffice

**Reply by szaimen** [reply, marked as answer]
I fear I am running out of ideas then. Maybe you can get some more help on https://help.nextcloud.com
